pkg/utils/k8sutils.go

```diff
@@ -60,6 +63,12 @@ const (
 	OcsClientTimeout = 10 * time.Second
 
 	OperatorVersionEnvVar = "OPERATOR_VERSION"
+
+	RookCephOperatorConfigName = "rook-ceph-operator-config"
+
+	CsiPluginTolerationKey = "CSI_PLUGIN_TOLERATIONS"
+
+	CsiProvisionerTolerationKey = "CSI_PROVISIONER_TOLERATIONS"
```

Comment:
As this is a temporary solution, let's make all of the changes inlined in the controller file. No need for consts